schemas/json/block.json

```
@@ -737,6 +737,17 @@
 		"render": {
 			"type": "string",
 			"description": "Template file loaded on the server when rendering a block."
+		},
+		"__experimentalAutoInsert": {
```

We've been avoiding adding `__experimental` properties to the schema to disincentivize using them until they are stabilized.

---

Thank you! To clarify, does that mean I should remove the `__experimental` prefix?

```suggestion
		"autoInsert": {
```

Or are you suggesting to remove the field from the schema definition altogether? I thought I need to specify the field in the schema or it won't be picked up from my blocks' `block.json` files... 🤔

---

Ah, looks like we don't need the field to be present in the schema for 3rd party blocks to work. I'll go ahead and remove it then 👍

---

4591c661d8